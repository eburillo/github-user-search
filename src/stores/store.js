import {createStore} from 'redux';
import ReposReducer from '../reducers/repos-reducer.js';

const store = createStore(ReposReducer);

export default store;
