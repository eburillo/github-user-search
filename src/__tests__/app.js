import React from 'react';
import ReactDOM from 'react-dom';
import {Provider} from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../stores/store.js';
import App from '../app';

describe('Render App', function() {
  it('renders without crashing', () => {
    const tree = renderer.create(
      <Provider store={store}>
    		<App />
    	</Provider>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
