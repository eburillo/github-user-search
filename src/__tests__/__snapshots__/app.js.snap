exports[`Render App renders without crashing 1`] = `
<div
  className="app">
  <div
    className="app-header">
    <h2>
      Github User Search
    </h2>
  </div>
  <div
    className="app-content">
    <form
      onSubmit={[Function]}>
      <h3>
        Search all repositories for a gitHub user
      </h3>
      <label
        className="input-wrapper">
        <input
          className="search-form-input"
          placeholder="Introduce a username"
          type="text" />
      </label>
      <button
        className="search-button"
        type="submit">
        Search
      </button>
    </form>
    <p>
      
    </p>
    <ul
      className="repo-list" />
  </div>
</div>
`;
